packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -274,36 +274,32 @@ export async function willForceUpload(
             )
         );
     }
-    vscode.window
-        .showInformationMessage(
-            localize("saveFile.info.confirmUpload", "Would you like to overwrite the remote file?"),
-            localize("saveFile.overwriteConfirmation.yes", "Yes"),
-            localize("saveFile.overwriteConfirmation.no", "No")
-        )
-        .then(async (selection) => {
-            if (selection === localize("saveFile.overwriteConfirmation.yes", "Yes")) {
-                const uploadResponse = Gui.withProgress(
-                    {
-                        location: vscode.ProgressLocation.Notification,
-                        title,
-                    },
-                    () => {
-                        return uploadContent(node, doc, remotePath, profile, binary, null, returnEtag);
+    Gui.infoMessage(localize("saveFile.info.confirmUpload", "Would you like to overwrite the remote file?"), {
+        items: [localize("saveFile.overwriteConfirmation.yes", "Yes"), localize("saveFile.overwriteConfirmation.no", "No")],
```

Comment:
I'm hoping that some day we reduce the duplication of string in our i18n files 😋 
ideally we will have some sort of shared strings file where things like `yes` and `no` will be included.

I'm not asking for this to be changed in this PR though 😋